app/logging/LoggerName.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package logging
+
+import play.api.Logger
+
+trait Logging extends LoggerName {
+
+  final val logger: Logger = Logger(loggerName)
+
+}
```

Comment:
I think this functionality might already be provided by Play as https://www.playframework.com/documentation/2.7.x/api/scala/play/api/Logging.html